### PR TITLE
refactored builders and cleanup

### PR DESCRIPTION
**Pull request by bergerdenes**
See detailed description in the commit message.

### CODE REVIEW

File: environment/src/test/java/com/sequenceiq/environment/environment/validation/network/AzureEnvironmentNetworkValidatorTest.java [hunk 1]
```diff
@@ -295,7 +295,7 @@ void testValidateDuringRequestWhenNoNetworkCidrAndNetworkId() {
 
     @Test
     void testValidateWhenNoNetworkCidrAndNoNetworkId() {
-        NetworkDto networkDto = NetworkTestUtils.getNetworkDto(new AzureParams(), null, null, null, null, 1);
+        NetworkDto networkDto = NetworkTestUtils.getNetworkDto(AzureParams.builder().build(), null, null, null, null, 1);
```
Comment (doktoric) [inline environment/src/test/java/com/sequenceiq/environment/environment/validation/network/AzureEnvironmentNetworkValidatorTest.java:298]: builder -> Builder ?

File: environment/src/test/java/com/sequenceiq/environment/environment/validation/network/YarnEnvironmentNetworkValidatorTest.java [hunk 2]
```diff
@@ -24,7 +24,7 @@ void setUp() {
 
     @Test
     void testValidateWhenNoNetworkCidrAndNoNetworkId() {
-        NetworkDto networkDto = NetworkTestUtils.getNetworkDto(null, null, new YarnParams(), null, null, 1);
+        NetworkDto networkDto = NetworkTestUtils.getNetworkDto(null, null, YarnParams.builder().build(), null, null, 1);
```
Comment (doktoric) [inline environment/src/test/java/com/sequenceiq/environment/environment/validation/network/YarnEnvironmentNetworkValidatorTest.java:27]: builder -> Builder ?

File: environment/src/test/java/com/sequenceiq/environment/environment/validation/network/YarnEnvironmentNetworkValidatorTest.java [hunk 3]
```diff
@@ -34,7 +34,7 @@ void testValidateWhenNoNetworkCidrAndNoNetworkId() {
 
     @Test
     void testValidateDuringFlowWhenNoQueueInYarnParams() {
-        NetworkDto networkDto = NetworkTestUtils.getNetworkDto(null, null, new YarnParams(), null, null, 1);
+        NetworkDto networkDto = NetworkTestUtils.getNetworkDto(null, null, YarnParams.builder().build(), null, null, 1);
```
Comment (doktoric) [inline environment/src/test/java/com/sequenceiq/environment/environment/validation/network/YarnEnvironmentNetworkValidatorTest.java:37]: builder -> Builder ?

File: environment/src/test/java/com/sequenceiq/environment/environment/validation/securitygroup/AwsEnvironmentSecurityGroupValidatorTest.java [hunk 4]
```diff
@@ -179,8 +179,7 @@ private SecurityAccessDto getNewSecurityAccessDto() {
     }
 
     private NetworkDto getNetworkDto(String vpcId) {
-        AwsParams awsParams = new AwsParams();
-        awsParams.setVpcId(vpcId);
+        AwsParams awsParams = AwsParams.builder().withVpcId(vpcId).build();
```
Comment (doktoric) [inline environment/src/test/java/com/sequenceiq/environment/environment/validation/securitygroup/AwsEnvironmentSecurityGroupValidatorTest.java:182]: builder -> Builder ?

File: environment/src/test/java/com/sequenceiq/environment/environment/validation/network/AzureEnvironmentNetworkValidatorTest.java (hunk 1 above)
Comment (bergerdenes) [reply environment/src/test/java/com/sequenceiq/environment/environment/validation/network/AzureEnvironmentNetworkValidatorTest.java:298]: No. `builder()` is a static method of the POJO that returns a builder object.

File: environment/src/test/java/com/sequenceiq/environment/environment/validation/network/YarnEnvironmentNetworkValidatorTest.java (hunk 2 above)
Comment (bergerdenes) [reply environment/src/test/java/com/sequenceiq/environment/environment/validation/network/YarnEnvironmentNetworkValidatorTest.java:27]: no

File: environment/src/test/java/com/sequenceiq/environment/environment/validation/securitygroup/AwsEnvironmentSecurityGroupValidatorTest.java (hunk 4 above)
Comment (bergerdenes) [reply environment/src/test/java/com/sequenceiq/environment/environment/validation/securitygroup/AwsEnvironmentSecurityGroupValidatorTest.java:182]: no

**Review by doktoric — APPROVED** [review]

**Pull request merged**